### ISSUE TITLE
The cls acc and loss is always 0.693147 and 87.3365?????

### ISSUE THREAD

**Issue by yungs2017** [CongWeilin/mtcnn-caffe#57]
When training the 12net, the cls acc is always 0.693147 or 87.3365, here some log 
```
I0912 17:29:04.343519 10590 solver.cpp:218] Iteration 98000 (541.938 iter/s, 0.922615s/500 iters), loss = nan
I0912 17:29:04.343559 10590 solver.cpp:237]     Train net output #0: cls_Acc = 0
I0912 17:29:04.343571 10590 solver.cpp:237]     Train net output #1: cls_loss = 0.693147 (* 1 = 0.693147 loss)
I0912 17:29:04.343580 10590 solver.cpp:237]     Train net output #2: pts_loss = 0 (* 0.5 = 0 loss)
I0912 17:29:04.343585 10590 solver.cpp:237]     Train net output #3: roi_loss = nan (* 0.5 = nan loss)
I0912 17:29:04.343595 10590 sgd_solver.cpp:105] Iteration 98000, lr = 0.256
I0912 17:29:05.266180 10590 solver.cpp:218] Iteration 98500 (541.948 iter/s, 0.922598s/500 iters), loss = nan
I0912 17:29:05.266221 10590 solver.cpp:237]     Train net output #0: cls_Acc = 0.296875
I0912 17:29:05.266233 10590 solver.cpp:237]     Train net output #1: cls_loss = 87.3365 (* 1 = 87.3365 loss)
I0912 17:29:05.266242 10590 solver.cpp:237]     Train net output #2: pts_loss = 0 (* 0.5 = 0 loss)
I0912 17:29:05.266250 10590 solver.cpp:237]     Train net output #3: roi_loss = 0 (* 0.5 = 0 loss)
I0912 17:29:05.266258 10590 sgd_solver.cpp:105] Iteration 98500, lr = 0.256
I0912 17:29:06.211555 10590 solver.cpp:218] Iteration 99000 (528.925 iter/s, 0.945313s/500 iters), loss = nan
I0912 17:29:06.211596 10590 solver.cpp:237]     Train net output #0: cls_Acc = 0
I0912 17:29:06.211609 10590 solver.cpp:237]     Train net output #1: cls_loss = 0.693147 (* 1 = 0.693147 loss)
I0912 17:29:06.211616 10590 solver.cpp:237]     Train net output #2: pts_loss = nan (* 0.5 = nan loss)
I0912 17:29:06.211624 10590 solver.cpp:237]     Train net output #3: roi_loss = 0 (* 0.5 = 0 loss)
I0912 17:29:06.211633 10590 sgd_solver.cpp:105] Iteration 99000, lr = 0.256
I0912 17:29:07.136104 10590 solver.cpp:218] Iteration 99500 (540.842 iter/s, 0.924485s/500 iters), loss = nan
```
what happen???
